README.md

```diff
@@ -35,13 +35,13 @@ dojo create [app] --name <appName>
 
 `@dojo/cli-create-app` generates a skeleton project structure for Dojo 2 into a directory using the the `--name` argument provided.
 
-All the dependencies have been pre-installed using `npm` including two `@dojo/cli` commands; [`@dojo/cli-build-webpack`](https://github.com/dojo/cli-build) and [`@dojo/cli-test-intern`](https://github.com/dojo/cli-test-intern).
+All the dependencies have been pre-installed using `npm` including two `@dojo/cli` commands; [`@dojo/cli-build-app`](https://github.com/dojo/cli-build-app) and [`@dojo/cli-test-intern`](https://github.com/dojo/cli-test-intern).
```

Comment:
All the dependencies have been pre-installed -> All the dependencies are pre-installed